space-cli/utils/file.go

```diff
@@ -34,10 +38,119 @@ func AppendConfigToDisk(specObj *model.SpecObject, filename string) error {
 	// Create a new file with out specs
 	return ioutil.WriteFile(filename, data, 0755)
 }
-func fileExists(filename string) bool {
+
+// ReadSpecObjectsFromFile returns the spec objects present in the file
+func ReadSpecObjectsFromFile(fileName string) ([]*model.SpecObject, error) {
+	var specs []*model.SpecObject
+
+	// Read the file first
+	data, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
what if the file content is empty ? check if len(data) == 0

Reply:
Doesn't matter. Checking for length below